### PR TITLE
Add new URL encode cheatsheet

### LINKED ISSUE #1282: Replace legacy "url escape" image with newer Cheat Sheet
We currently show an image in the results for the query [url escape](https://duckduckgo.com/?q=url+escape).

We should showcase this information with a cheat sheet instead. Let's ditch the image!

### PR DESCRIPTION
**Pull request by gaulrobe**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [x] New Instant Answer
    - [x] Cheat Sheets: **`New URL Encode Cheat Sheet`**

###### Description of changes

New URL Encode cheat sheet, Finishes up the work began in #2564. For now, I only went to %7F since those are the most commonly used characters.

###### Which issues (if any) does this fix?

Fixes #1282 - now we will have a URL cheat sheet instead of an image!

###### People to notify (@mention interested parties)

@moollaza @SanketDG

---

Instant Answer Page: https://duck.co/ia/view/url_encode_cheat_sheet

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @gaulrobe  (I volunteer to be maintainer if @SanketDG no longer wishes to do so)

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/url-encode.json [hunk 1]
```diff
@@ -0,0 +1,420 @@
+{
+   "id":"url_encode_cheat_sheet",
+   "name":"URL Encoding CheatSheet",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/url-encode.json]: Lets remove " cheatsheet " from name

File: share/goodie/cheat_sheets/json/url-encode.json [hunk 2]
```diff
@@ -0,0 +1,420 @@
...
+   "description":"CheatSheet for URL encoding",
+   "metadata":{
+      "sourceName":"ASCII Encoding Reference",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/url-encode.json]: Change source name to  "w3schools "

**Comment by gaulrobe** [comment]
@jophab thanks for the suggestions. I cleaned up some of the namings :+1:

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=url encoding cheatsheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/url_encode_cheat_sheet_mobile.png?SfehluCRJN)

**Comment by SanketDG** [comment]
@gaulrobe Thank you so much, I appreciate the efforts. I haven't been able to get a lot of time lately.

**Comment by gaulrobe** [comment]
@SanketDG you are very welcome, glad to have helped!

**Comment by moollaza** [comment]
@gaulrobe @SanketDG Thanks! This LGTM :+1:

@gaulrobe I'll set you to the maintainer 👍

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=url encoding cheatsheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/url_encode_cheat_sheet_mobile.png?QzpdQfNrSv)

**Comment by daxtheduck** [comment]
@gaulrobe this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

**Comment by daxtheduck** [comment]
@gaulrobe this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

**Comment by edgesince84** [comment]
@SanketDG I'm reaching out to let you know that we recently launched a new forum with the initiative of improving programming-related searches, and I hope you'll [join us](https://forum.duckduckhack.com/) if you get some more time to contribute soon!

There are a lot of important tasks up for grabs that we'd like your help with, so feel free to and [jump in](https://forum.duckduckhack.com/t/duckduckhack-programming-mission-overview/53).

**Pull request merged**